src/platforms/hosted/gdb_if.c

```diff
@@ -35,8 +35,12 @@
 typedef SOCKET socket_t;
 #define PRI_SOCKET "zu"
 #ifndef __CYGWIN__
+#ifdef _WIN64
 typedef signed long long ssize_t;
-#endif
+#else
+typedef int ssize_t;
```

Comment:
Please make this explicitly signed, for platform reasons involving the ability to set the designed signed-ness of numbers to unsigned in MSVC. That way this won't break if someone tries that.